ccd-definition/ComplexTypes/IdamUserDetails.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
I wonder if we should model this slightly differently... 
In my mind the technical data we pull from IDAM (email & id) should be stored in an IDAM specific complex type e.g., `applicantSolicitor1IdamUserDetails` while the business data we care about from a business point of view should be stored in a complex type that doesn't contain any technical data specific to IDAM: `applicantSolicitor1EmailAddress` .The `applicantSolicitor1IdamUserDetails` data  should only be kept on the case for a relatively short amount of time: (1) clear `applicantSolicitor1IdamUserDetails.email` as soon as we no longer need it, i.e. when it has been copied across into `applicantSolicitor1EmailAddress.email`. (2) clear `applicantSolicitor1IdamUserDetails.id` in Camunda immediately after case roles have been fixed up (@dharmendrak story would take care of that).